### DISCUSSION TITLE
Al utilizar Stack tenemos un componente hijo que admita grow y shrink?

### DISCUSSION THREAD

**Discussion by NicoCroce** [TiendaNube/nimbus-design-system#65] [Q&A] [answered]
Puedo utilizar el componente Box alterando width para ese caso? 

https://tiendanube.github.io/nimbus-design-system/?path=/docs/atomic-stack--flex

**Comment by juanchigallego** [comment, marked as answer]
Hola @NicoCroce !

Para ese caso, podés anidar componentes `<Stack>`, usando `display="flex"` en el padre y `display="inline-flex"` ó `"flex"` en los hijos, y usando la propiedad `flex` para definir el grow y shrink.

Te dejo [un sandbox](https://codesandbox.io/s/relaxed-stonebraker-p5tf8h?file=/src/App.tsx) de ejemplo para demostrar estas propiedades.

Cualquier duda avisame!
